test/cpp/cpp_test_util.h

```diff
@@ -38,11 +37,13 @@ static inline void AllClose(at::Tensor tensor, at::Tensor xla_tensor,
 
 static inline void AllClose(at::Tensor tensor, XLATensor& xla_tensor,
                             double rtol = 1e-5, double atol = 1e-8) {
-  EXPECT_TRUE(CloseValues(tensor, ToTensor(xla_tensor), rtol, atol));
+  EXPECT_TRUE(CloseValues(tensor, xla_tensor.ToTensor(), rtol, atol));
 }
 
 void ForEachDevice(const std::function<void(const Device&)>& devfn);
```

Comment:
Do we still need this method? I'd try to remove it.

Reply:
Please don't, `test_tensor.cpp` still uses it.